mailpoet/lib/EmailEditor/Engine/Renderer/Renderer.php

```diff
@@ -45,6 +45,20 @@ public static function getTheme() {
     return self::$theme;
   }
 
+  public static function maybeConvertPreset($value, $variables) {
```

Comment:
Does this need to be `public static`? I see usage only in this class, so what about making it private? 

I also see it is needed because `$this->themeController->getStyles` may return some variables in the "compressed" format. The `ThemeController:getStyles` is also used in the ContentRenderer to pass styles to preprocessors. These are the only two usages and both are in the renderer. What about moving this `maybeConvertPreset` to the ThemeController? If we want to retain the `getStyles` to return exactly what is set we could introduce another `ThemeController:getStylesWithValidPresets` method that would return styles with valid variable names? What do you think?

Reply:
Handled in https://github.com/mailpoet/mailpoet/pull/5572/commits/24ce2a8fe30524ff2aaa83ff5b723f3ee8918d91